openlibrary/templates/lib/nav_foot.html

```diff
@@ -41,7 +41,7 @@ <h2>$:_('Develop')</h2>
         <h2>$:_('Help')</h2>
         <ul>
           <li><a href="/help">$_('Help Center')</a></li>
-          <li><a href="/contact?path=$request.path" title="$_('Problems')">$_('Report A Problem')</a></li>
+          <li><a href="/contact?path=$request.fullpath" title="$_('Problems')">$_('Report A Problem')</a></li>
```

Comment:
This should be url escaped now, since `fullpath` could contain things like `?` or `&`
```suggestion
          <li><a href="/contact?$:urlencode(dict(path=request.fullpath))" title="$_('Problems')">$_('Report A Problem')</a></li>
```